<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Letter Frequency</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="freqTable">
      <row>
       <property name="text">
        <string>1</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>2</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>3</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>4</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>5</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>6</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>7</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>8</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>9</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>10</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>11</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>12</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>13</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>14</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>15</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>16</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>17</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>18</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>19</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>20</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>21</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>22</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>23</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>24</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>25</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>26</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>Plaintext</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Ciphertext</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Frequency</string>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string>A</string>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string>B</string>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string>C</string>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string>D</string>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string>E</string>
       </property>
      </item>
      <item row="5" column="0">
       <property name="text">
        <string>F</string>
       </property>
      </item>
      <item row="6" column="0">
       <property name="text">
        <string>G</string>
       </property>
      </item>
      <item row="7" column="0">
       <property name="text">
        <string>H</string>
       </property>
      </item>
      <item row="8" column="0">
       <property name="text">
        <string>I</string>
       </property>
      </item>
      <item row="9" column="0">
       <property name="text">
        <string>J</string>
       </property>
      </item>
      <item row="10" column="0">
       <property name="text">
        <string>K</string>
       </property>
      </item>
      <item row="11" column="0">
       <property name="text">
        <string>L</string>
       </property>
      </item>
      <item row="12" column="0">
       <property name="text">
        <string>M</string>
       </property>
      </item>
      <item row="13" column="0">
       <property name="text">
        <string>N</string>
       </property>
      </item>
      <item row="14" column="0">
       <property name="text">
        <string>O</string>
       </property>
      </item>
      <item row="15" column="0">
       <property name="text">
        <string>P</string>
       </property>
      </item>
      <item row="16" column="0">
       <property name="text">
        <string>Q</string>
       </property>
      </item>
      <item row="17" column="0">
       <property name="text">
        <string>R</string>
       </property>
      </item>
      <item row="18" column="0">
       <property name="text">
        <string>S</string>
       </property>
      </item>
      <item row="19" column="0">
       <property name="text">
        <string>T</string>
       </property>
      </item>
      <item row="20" column="0">
       <property name="text">
        <string>U</string>
       </property>
      </item>
      <item row="21" column="0">
       <property name="text">
        <string>V</string>
       </property>
      </item>
      <item row="22" column="0">
       <property name="text">
        <string>W</string>
       </property>
      </item>
      <item row="23" column="0">
       <property name="text">
        <string>X</string>
       </property>
      </item>
      <item row="24" column="0">
       <property name="text">
        <string>Y</string>
       </property>
      </item>
      <item row="25" column="0">
       <property name="text">
        <string>Z</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="analyzeButton">
        <property name="text">
         <string>Analyze</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="closeButton">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
